src/routes/assemblies/[id]/members/[groupby]/+page.svelte

```diff
@@ -1,8 +1,243 @@
 <script lang="ts">
+	import Header from '$components/Assemblies/Members/Header.svelte';
+	import Tab from '$components/Assemblies/Members/Tab.svelte';
+	import PoliticianProfile from '$components/PoliticianProfile/PoliticianProfile.svelte';
+	import {
+		Accordion,
+		AccordionItem,
+		Button,
+		Checkbox,
+		FormGroup,
+		Search
+	} from 'carbon-components-svelte';
+	import Minimize from 'carbon-icons-svelte/lib/Minimize.svelte';
+	import SearchLocate from 'carbon-icons-svelte/lib/SearchLocate.svelte';
+	import scrollama from 'scrollama';
+	import { onMount } from 'svelte';
+	import type { PoliticianGroupBy } from './+page.js';
+
 	export let data;
-	$: ({ groups } = data);
+	$: ({ groups, groupByTabs } = data);
+	$: currentPath = groupByTabs.find((e) => e.isActive)?.path ?? '';
```

Comment:
Usually `e` is for the event, isn't it? I think you can just deconstructed `({isActive}) => isActive`